package.json

```diff
@@ -29,11 +29,14 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.26.0",
+    "rxjs": "^5.5.0",
```

Comment:
Does it really also have to be declared as a dev dependency? Just asking to learn ;-)

Reply:
This library is intended to be used as a complimentary package for `rxjs` so we expect that user will have it installed (that's why we declare it as "peerDependency"). However, we need `rxjs` for testing purposes - that's why it's "devDependency".